flipper-plugin-realm/src/components/DataTable.tsx

```diff
@@ -79,10 +88,12 @@ export const DataTable = ({
   setNewInspectionData,
   enableSort,
   hasMore,
-  totalObjects,
-  fetchMore,
+  totalObjects = 0,
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  fetchMore = () => {},
```

Comment:
We should probably just get rid of this rule (I don't see the point), but here's an implementation that passes the linter:
```suggestion
  fetchMore = () => undefined,
```